zip-0216.rst

```diff
@@ -200,32 +239,44 @@ Security and Privacy Considerations
 
 This ZIP eliminates a potential source of consensus divergence between differing full node
 implementations. At the time of writing (February 2021), no such divergence exists for any
-production implementation of Zcash, but the alpha-stage `zebrad` node implementation would
-be susceptible to this issue.
+production implementation of Zcash, but an early alpha version of the `zebrad` node
+implementation would have been susceptible to this issue.
 
 
 Deployment
 ==========
 
-This ZIP is proposed to activate with Network Upgrade 5. Requirements on points encoded in
-payment addresses and full viewing keys MAY be enforced in advance of NU5 activation.
+This ZIP activated with Network Upgrade 5. Requirements on points encoded in payment
+addresses and full viewing keys MAY be enforced in advance of NU5 activation.
+
+`zcashd` PR #6399 [#zcashd-pr6399]_ retroactively enforces canonical encoding of Jubjub
+points for the entire chain history, as described in the `Retroactive applicability`_
+section.
```

Comment:
Either mention https://github.com/zcash/zcash/pull/6000 as well, or delete this.